File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.cpp
class Solution {
public:
    int finalValueAfterOperations(vector<string>& operations) {
        int X=0;
        for(const std::string& v:operations){
            if(v == "++X" || v == "X++"){
                X++;
            }else if(v == "--X" || v == "X--"){
                X--;
            }
        }
        return X;
    }
};

File: 2137-final-value-of-variable-after-performing-operations/final-value-of-variable-after-performing-operations.py
class Solution(object):
    def finalValueAfterOperations(self, operations):
        """
        :type operations: List[str]
        :rtype: int
        """
        X =0
        for v in operations:
            if v == "++X" or v == "X++":
                X+= 1
            elif v == "X--" or v == "--X":
                X-=1
            
        return X